study.md

```diff
@@ -147,5 +160,5 @@ using `hank` that equals "My name is Hank Hill and I'm a propane and propane
 accesories salesman that enjoys drinking beer.".
 
 ```javascript
-// your answer here
+`My name is ${hank['given_name']} and I'm a ${hank['hobbies'][1]} that enjoys ${hank['hobbies'][0]}`;
```

Comment:
This will print "My name is Hank and I'm a propane and propane accessories that enjoys drinking beer".